### PR TITLE
239 SIM-FE-Create accounts in the frontend with its pair of keys

### PR DESCRIPTION
**Pull request by bradleySuira**
This PR includes a simple implementation to use local generated accounts in the front end using viem library 
- we store now a random private key in `accountsStore.privateKeys` local storage
- we generate the accounts based on the private key stored
- we show the address of the account related to the pk stored

### CODE REVIEW

File: frontend/src/utils/accounts.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts'
+
+export const getAccount = (privateKey: `0x${string}`) => {
+    return privateKeyToAccount(privateKey)
+}
+
+export const getPrivateKey = () => {
```
Comment (cristiam86) [inline frontend/src/utils/accounts.ts:7]: @bradleySuira should these wrappers go on "utils"?
generate looks like a better name for this one and getAccountFromPrivatekey also looks better in my opinion for the other method

File: frontend/src/views/ProfileView.vue [hunk 2]
```diff
@@ -31,9 +31,9 @@ const handleShowSetDefaultAccount = (address: string) => {
 const handleHideSetDefaultAccount = (address: string) => {
   showSetDefaultAccount.value[address] = false
 }
-const setCurentUserAddress = (address: string) => {
+const setCurentUserAddress = (address: `0x${string}`) => {
   if (address) {
-    store.currentUserAddress = address
+    store.currentPrivateKey = address
```
Comment (cristiam86) [inline frontend/src/views/ProfileView.vue:36]: @bradleySuira, how are you assigning an address to a private key?

**Review by cristiam86 — CHANGES_REQUESTED** [review]
Would you also remove the create account endpoint from the backend?

File: frontend/src/utils/accounts.ts (hunk 1 above)
Comment (bradleySuira) [reply frontend/src/utils/accounts.ts:7]: for now in utils but we can define later if we need more related to wallet restore and similar the name is now your suggestion your are right

File: frontend/src/views/ProfileView.vue (hunk 2 above)
Comment (bradleySuira) [reply frontend/src/views/ProfileView.vue]: done, that's because we used address in the past

**Comment by bradleySuira** [comment]
> Would you also remove the create account endpoint from the backend? Also, please wait until the refactor is merged and change the destination of this PR to staging

for now I removed the request from the frontend and json rpc service, in the backend we have some dependencies at least in tests that needs the create account call for example when fund accounts, maybe we need to define how to create accounts in the same format like in frontend but in the backend then can be removed completely

**Comment by cristiam86** [comment]
@bradleySuira please update from staging and solve conflicts

File: tests/common/response.py [hunk 3]
```diff
@@ -1,4 +1,6 @@
 def assert_dict_struct(data, structure):
+    print("data", data)
```
Comment (cristiam86) [inline tests/common/response.py:2]: @bradleySuira please remove this prints :)

**Review by cristiam86 — CHANGES_REQUESTED** [review]

File: tests/common/response.py (hunk 3 above)
Comment (bradleySuira) [reply tests/common/response.py]: fixed

**Review by cristiam86 — APPROVED** [review]

**Pull request merged**